.vscode/settings.json

```diff
@@ -1,17 +1,3 @@
 {
-  "typescript.tsdk": "node_modules/typescript/lib",
-  "typescript.enablePromptUseWorkspaceTsdk": true,
-  "search.exclude": {
-    "**/.git": true,
-    "**/node_modules": true,
-    "**/.next": true,
-    "**/vendor": true,
-    "**/lib": true,
-    "**/.mirrorful": true,
-    "**/dist": true
-  },
-  "cSpell.words": ["Mirrorful"],
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
-  }
-}
+"isGitignoreExtActive": true,
```

Comment:
Just curious @Bismay5467 why did you change the settings? I'm going to re-instate these.